### PR TITLE
updates for WDIDC7

### PR DESCRIPTION
**Pull request by adambray**

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Nice! :+1:

File: evaluation.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 ### Back-end
 
-- Your API includes at least 2 models, one of which may be a user
-- Your models include at least one association
+- Your API includes at least 2 models, backed by MongoDB (one of which may be a user)
+- Your models include at some form of 'association', e.g. nested documents
```
Comment (jshawl) [inline evaluation.md:9]: feel like user auth should be required either via 3rd part api or local auth - we're teaching 3 classes on it

File: evaluation.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ well-commented.
 
 ### Submission
 
-- One person from your team should file an issue on the [pbj-project3-gallery repo](https://github.com/ga-dc/pbj-project3-gallery). The issue should contain:
+- One person from your team should file an issue on the [project3-gallery repo](https://github.com/ga-dc/project3-gallery). The issue should contain:
```
Comment (jshawl) [inline evaluation.md:31]: should this be namespaced to wdi7? legit question - no strong opinions from me

File: readme.md [hunk 3]
```diff
@@ -41,26 +45,30 @@ Besides finishing WDI, you surely have one or two things you'd love to do with y
...
+Attendance will be taken on the following dates/times:
+
+| Date | Time |
+|------|------|
+| Wednesday, Dec 2 | 9:00am  |
+| Thursday, Dec 3  | 9:00am  |
+| Friday, Dec 4    | 10:00am |
+| Monday, Dec 7    | 10:00am |
+| Tuesday, Dec 8   | 9:00am  |
+| Friday, Dec 11   | 12:30am |
```
Comment (jshawl) [inline readme.md]: how come? anything going on those days? also 12:30pm on friday, not am

File: evaluation.md (hunk 2 above)
Comment (adambray) [inline evaluation.md:31]: I vote no... moving forward, we'll have to make fewer edits to readmes

File: readme.md (hunk 3 above)
Comment (adambray) [inline readme.md]: Lessons, outcomes, or presentations

Comment (adambray) [inline readme.md]: And Monday needs to be 9am as well now

**Pull request merged**